src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -332,7 +332,7 @@
     <value>The package meta-data locale</value>
   </data>
   <data name="PackageUrl_KeywordDescription" xml:space="preserve">

```

Comment:
Do we need examples in the resources for all of these as we do for the required properties? (@denelon / @KevinLaMS)

Reply:
I think it would be good to have examples, but I'd want to walk the UX to see how it looks. Some items are easier than others. The ones that are an enumeration might also be tricky like "user" & "machine" for scope.

Reply:
I think we should be careful adding too many examples, it could overload the screen and feel overwhelming since there will be quite a lot of text. I think this is a good question to figure out during our bug bash